src/ucp/proto/proto_multi.c

```diff
@@ -41,6 +41,10 @@ ucs_status_t ucp_proto_multi_init(const ucp_proto_multi_init_params_t *params,
     ucs_assert(params->max_lanes >= 1);
     ucs_assert(params->max_lanes <= UCP_PROTO_MAX_LANES);
 
+    if (!ucp_proto_init_check_err_handling(&params->super.super)) {
```

Comment:
IMO we should pass err handling support as part of https://github.com/openucx/ucx/blob/d7a1b4527dd05e37a1f4630ed99142c8c41eaa93/src/ucp/proto/proto_common.h#L113

instead of checking by function pointer.
we can assert by function pointer if needed